convertSortedListToBST.py

```diff
@@ -0,0 +1,44 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+class Solution:
+    def sortedListToBST(self, head):
+        """
+        :type head: ListNode
+        :rtype: TreeNode
+        """
+        if not head:
+            return None
+        count = 0
+        curNode = head
+        while curNode:
```

Comment:
As discussed offline, please use two pointers
```
while fast and fast.next:
    fast = fast.next.next
    slow = slow.next
```
When `fast` is `None`, `slow` should be at the centre of the list.